app/src/main/java/com/quran/labs/androidquran/util/ShareUtil.java

```diff
@@ -53,29 +53,51 @@ public void shareViaIntent(Activity activity, String text, @StringRes int titleR
     activity.startActivity(Intent.createChooser(intent, activity.getString(titleResId)));
   }
 
-  public String getShareText(Context context,
+  public String getShareTextperTranslation(Context context,
                                     QuranAyahInfo ayahInfo,
-                                    String[] translationNames) {
+                                    String[] translationNames,String selectedTranslation) {
```

Comment:
Space after comma. Also, if followed the existing style, I'd move down the last argument into new line.